documentation/directory-structure.md

```diff
@@ -0,0 +1,197 @@
+|            |   \---preview
+|            |       \---2017-04-24-preview
+|            |           \---examples
+|            \---readme.md
+```
+
+In the example above:
+
+`rp` folders are `automation`, `batch`, `playfab`.  
+TypeSpec folders are `specification/playfab/Playfab`.  
+`rpnamespace` names present are: `Microsoft.Automation` (in `automation` rp), `Microsoft.Batch` (in `batch` rp) and `Microsoft.Playfab` (in `playfab` rp).
+
+## Folder Structure for service groups
+
+In some existing cases the `specification/rp` folder is not a root of one resource provider,
+but of a **service group**. A service group combines multiple resource providers,
+from multiple teams, in one `specification/rp` folder. In this situation the layout
+constraints described in this article are partially violated, e.g. given `rpnamespace` instead
+of having `stable` and `preview` as direct children, has extra layer of "sub-service" name, like `rpnamespace/compute`. This scenario is not allowed going forward and existing teams will migrate away from it, eventually.
+
+Examples of an existing services groups: 
+- [`specification/compute`]
+- [`specification/workloads`]
+
+## common-types
+
+Specification files and AutoRest configuration files in one resource provider folder are better to refer to files in the same resource provider folder. Entity type definition that can be shared cross resource providers or services should to be placed and maintained either under the folder [**common-types**](https://github.com/Azure/azure-rest-api-specs#common-types) under specification, or under **common-types** folder of service group folder structure. The former supports the entity type sharing cross rp folders, while the latter supports the entity type sharing cross components or services under the same rp folder.
+
+Refer to [Resource-Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/common-types/resource-management) common types for example.
+
+## FAQ
+
+### Should my service team have one or multiple resource provider folders?
+
+Should given service team add multiple `specification/rp` folders, or just one?
+It depends on the following considerations:
+    
+- An rp folder leads to a separate SDK package. Is it expected to have separate SDK packages for different services?
+- Service in one folder share the same versioning cycle. Can service in one folder share the same version label, and upgrade together in the future?
+- Specification files and AutoRest configuration files in one rp folder are better to refer to files in the same rp folder. Note: Entity type definition that needs to be referred cross rp folders should be placed and maintained under the folder [**common-types**](https://github.com/Azure/azure-rest-api-specs#common-types).
+- For more considerations, you may consult the reviewer in API design review. To initiate the review, Please submit an [Azure SDK intake questionnaire](https://aka.ms/sdk-apex).
```

Comment:
@maririos is this link still up to date?

Reply:
ohh interesting. It is not. Good catch :)
My advice will be to go to the Office hours in https://eng.ms/docs/products/azure-developer-experience/design/api-design

Reply:
The new document doesn't mention this at all.